### PR TITLE
update Dockerfile for openshift CI

### PR DESCRIPTION
**Pull request by pgier**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+WORKDIR /go/src/github.com/prometheus/node_exporter
+COPY . .
+RUN make build
+
+FROM  openshift/origin-base
+
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
-COPY node_exporter /bin/node_exporter
+COPY --from=builder /got/src/github/prometheus/node_exporter/node_exporter /bin/node_exporter
```
Comment (simonpasquier) [inline Dockerfile]:
```suggestion
COPY --from=builder /go/src/github.com/prometheus/node_exporter/node_exporter /bin/node_exporter
```

Comment (pgier) [reply Dockerfile]: Thanks, should be fixed now.

**Pull request merged**